sqlmesh/dbt/adapter.py

```diff
@@ -311,7 +311,10 @@ def resolve_identifier(self, relation: BaseRelation) -> t.Optional[str]:
     def _map_table_name(
         self, database: t.Optional[str], schema: t.Optional[str], identifier: t.Optional[str]
     ) -> exp.Table:
-        name = ".".join(p for p in (database, schema, identifier) if p is not None)
+        name = normalize_model_name(
```

Comment:
is this call to `normalize_model_name` necessary? I think we'll get normalized identifiers when calling this?

cc @tobymao

Reply:
well it fixes the issue we're facing. i don't know if it's right